systemtest/src/main/java/io/strimzi/systemtest/resources/KubernetesResource.java

```diff
@@ -307,6 +312,84 @@ public static Service deployBridgeNodePortService(String bridgeExternalService,
             .endSpec().build();
     }
 
+    public static <T extends HasMetadata> NetworkPolicy applyNetworkPolicySettingsForResource(T resource, String name) {
```

Comment:
I think this is too wide. This will open the network even for the CO and therefore the tests will not really test that the CO is able to deal with this on its own. I think you need to make this more specific to allow only connections for the tests.

Reply:
AFAIU CO deal with NP by it's own in case that Kafka Connector is used. In that case we don't apply any NP changes and we communicate with Connect only via operator. I am not sure about mm2 and bridge atm. Is that what you mean we should do?

Reply:
But that is not right, or? That means that this test will test the known issues, but never found any new issues in other areas. AFAIK we have this problem repeatedly and we found about these issues only form user issues. So we need to make sure this test not only covers the known cases but has a chance to find new cases as well.